content/about/us.md

```diff
@@ -1,9 +1,25 @@
 ---
-title: "About us"
+title: "Developers"
 date: 2018-12-29T11:02:05+06:00
 layout: "overview"
 ---
 
-### Coming soon
+The GRASS GIS project is an international team effort with [scientists and developers](https://grasswiki.osgeo.org/wiki/Team) participating from various fields. GRASS - Geographic Resources Analysis Support System has been under continual development since 1982 and has involved a large number of federal US agencies, universities, and private companies. The core components of GRASS and the management of the integration of the efforts into GRASS releases were accomplished by the Construction Engineering Resesarch Laboratory (CERL) in Champaign, Illinois. It has been estimated that several million dollars of GRASS GIS software development efforts across the government have been completed since the 4.1 release. Since 1997 a worldwide network of developers continue to release new GRASS GIS versions (see GRASS history).
```

Comment:
```suggestion
The GRASS GIS project is an international team effort including [scientists and developers](https://grasswiki.osgeo.org/wiki/Team) from various fields. GRASS - Geographic Resources Analysis Support System - has been under continuous development since 1982 and has involved a large number of federal US agencies, universities, and private companies. The core components of GRASS and the management of the integration of the efforts into GRASS releases were accomplished by the Construction Engineering Resesarch Laboratory (CERL) in Champaign, Illinois. It has been estimated that several million dollars of GRASS GIS software development efforts across the government have been completed since the 4.1 release. Since 1997 a worldwide network of developers continue to release new GRASS GIS versions (see GRASS history).
```